src/main/kotlin/net/skripthub/docstool/modals/SyntaxData.kt

```diff
@@ -1,81 +1,53 @@
 package net.skripthub.docstool.modals
 
+import ch.njol.util.StringUtils
+
+@Suppress("ArrayInDataClass")
```

Comment:
This is a good call out. This shouldn't be part of this PR, but I can going to look at making this follow better data class practices in the future.